packages/@okta/vuepress-site/docs/reference/api/oie-policies/index.md

```diff
@@ -0,0 +1,737 @@
+| id       | String  | Unique identifier for this Policy (read-only).                                               |
+| name     | String  | Human-readable name for the Policy, configurable during creation or updating.                |
+| type     | String  | Type of the policy. For IdP Routing Policy objects, this needs to be `Okta:IdentifierMatch`. |
+| status   | String  | `ACTIVE`  or  `INACTIVE`.                                                                    |
+| default  | Boolean | `true` for the first instance of this policy, which gets created by default.                 |
+
+### Identifier Match Policy Object Example
+
+```json
+{
+    "id": "rst10y1rmKOjickDM0g4",
+    "name": "Default Policy",
+    "type": "Okta:IdentifierMatch",
+    "status": "ACTIVE",
+    "default": true,
+    "_links": {
+        "self": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10y1rmKOjickDM0g4",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        },
+        "rules": {
+            "href": "https://idx.okta1.com/api/v1/policies/rst10y1rmKOjickDM0g4/rules",
+            "hints": {
+                "allow": [
+                    "GET"
+                ]
+            }
+        }
+    }
+}
+```
+## Identifier Match Rule Object
+
+| Property    | Type                                                                                  | Description                                                                                                        |
+|-------------|---------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------|
+| name        | String                                                                                | Human-readable name for the Rule, configurable during creation or updating.                                        |
+| id          | String                                                                                | Unique identifier for this Rule (read-only).                                                                       |
+| type        | String                                                                                | Type of the policy. For Identifier Match Rule Objects, this needs to be `Okta:IdentifierMatch`.                    |
+| priority    | Integer                                                                               | Used to determine which rules take precedence.                                                                     |
+| conditions  | Array                                                                                 | No conditions are supported for this rule type, so this must be an empty array.                                    |
```

Comment:
`No conditions are supported for this rule type, so this must be an empty array.` -> `Identifier and App conditions are supported for this rule type.`